src/components/steps/Loading.vue

```diff
@@ -30,6 +31,7 @@ export default {
     sessionId(value) {
       if (value) {
         this.$router.push(`/wallet/transactions?session_id=${this.sessionId}`)
+        this.setDefaultTemplates({ locale: this.locale })
```

Comment:
IMO, this logic doesn't belong to the loading component. We can call this method after unlock the wallet in vuex. There we should also check if the wallet already has the default templates before save them.

Reply:
#1714